### ISSUE TITLE
Wrong title for lethal hits

### ISSUE THREAD

**Issue by MoscowskyAnton** [toadchild/40kdice#17]
Plot title is same (autowound on 6s) no matter what critical hit value I set for lethal hits 
![image](https://github.com/toadchild/40kdice/assets/49156277/33fec44b-ce1e-4ea9-a741-d5549d2b411c)

**Comment by toadchild** [comment]
Good call, that needs to either say "crits" generically or reflect what the selected critical value is.

Good news is that it's just a labeling issue, the actual probabilities reflect the selected value.
